front_end/shared.js

```diff
@@ -0,0 +1,13 @@
+//displaying start and due dates on the course table
+var days = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
```

Comment:
Unless there is a good reason to have this as a global variable (like if it's used by multiple functions), please move this variable inside the function. Even then, it would be better to declare it as a function rather than a global variable.